apps/web/test/lib/getSchedule.test.ts

```diff
@@ -1325,6 +1325,87 @@ describe("getSchedule", () => {
         }
       );
     });
+
+    test("check the availability of guest when rescheduling if guest is a cal.com user", async () => {
```

Comment:
![Screenshot 2024-09-22 114759](https://github.com/user-attachments/assets/d0ea70ae-4874-422f-ba35-84ec769551b8)

This test keeps failing for me locally, is there something I am supposed to set before running it?

Reply:
Fixed. Please check now.